override-THIRD-PARTY.properties

```diff
@@ -64,3 +64,5 @@ org.slf4j--jcl-over-slf4j--1.7.7=Apache-2.0
 xpp3--xpp3--1.1.3.4.O=BSD Variant (License for the Extreme! Lab)
 # https://web.archive.org/web/20190614191124/http://www.extreme.indiana.edu/dist/java-repository/xpp3/licenses/LICENSE.txt
 xpp3--xpp3--1.1.4c=BSD Variant (License for the Extreme! Lab)
+# https://projects.eclipse.org/projects/ee4j.jms
+jakarta--jms--jakarta.jms-api--2.0.3=EPL-2.0
```

Comment:
Please can we keep these alphabetical?  It would be great if we find time to create some simple tests for this project (or maybe use pre-commit to auto-order the files?)